diff --git a/build/node/esm/src/signing/chains/KyveSigner.js b/build/node/esm/src/signing/chains/KyveSigner.js
index 83d0c0a9748185fcba9779fa24c0a1a01c266eaa..eaa570c1152dfa46b23d0e908b0a51a6ee872de8 100644
--- a/build/node/esm/src/signing/chains/KyveSigner.js
+++ b/build/node/esm/src/signing/chains/KyveSigner.js
@@ -1,6 +1,6 @@
-import { EthereumSigner } from "./index.js";
+import EthereumSigner from "./ethereumSigner.js";
 import { SignatureConfig } from "../../constants.js";
 export default class KyveSigner extends EthereumSigner {
-    signatureType = SignatureConfig.KYVE;
+  signatureType = SignatureConfig.KYVE;
 }
 //# sourceMappingURL=KyveSigner.js.map
